Source/WebKit/Scripts/webkit/tests/GeneratedSerializers.h

```diff
@@ -73,6 +73,11 @@ namespace JSC { enum class Incredible; }
 namespace Testing { enum class StorageSize : uint8_t; }
 namespace WebCore { class ScrollingStateFrameHostingNode; }
 namespace WebCore { class ScrollingStateFrameHostingNodeWithStuffAfterTuple; }
+#if USE(CFBAR)
+#endif
```

Comment:
Can we avoid outputting this empty `#if USE(CFBAR)/#endif` code?

Would prefer to fix this before landing, but not strictly required to do so.

Reply:
What this patch doesn't make obvious is that this issue is long standing
If you looked at your <builddir>/DerivedSources/WebKit/GeneratedSerializers files you'd see tons of these repeated empty conditionals.

I agreed cleaning them would be great, but should be a standalone hack to the serializer script.